src/components/button/examples/button-disabled-vs-hidden.tsx

```diff
@@ -0,0 +1,160 @@
+import { Component, h, State } from '@stencil/core';
+import { ListSeparator, MenuItem } from 'src/interface';
+
+/**
+ * Hiding or Disabling Irrelevant Buttons in User Interfaces
+ *
+ */
+@Component({
+    tag: 'limel-example-button-disabled-vs-hidden',
+    shadow: true,
+    styleUrl: 'button-disabled-vs-hidden.scss',
+})
+export class ButtonDisabledVsHideExample {
+    @State()
+    private value: boolean = false;
+
+    @State()
+    private clicked: boolean = false;
+
+    @State()
+    public required: boolean;
+
+    private items: Array<ListSeparator | MenuItem> = [
+        { text: 'Edit', icon: 'save_as' },
```

Comment:
including an Edit option in the split button which is for Save is not a good design. I suggest creating a <div class="dont"> example in which the two buttons are available, but one of them is disabled.

Reply:
Also "Delete" would be a better example for what an admin may be allowed to do, but not a normal user

Reply:
> including an Edit option in the split button which is for Save is not a good design. I suggest creating a
> 
> example in which the two buttons are available, but one of them is disabled.

I think `Save` and `Save as` actually would be good show here. Especially since it's our real example from the webclient

Reply:
> I suggest creating a
> 
> example in which the two buttons are available, but one of them is disabled.

But what is the reason to have a disabled one, since for the user it will never be an active(or not disabled) button?

Reply:
to compare the two different **good** and **bad**, side by side.